src/main/java/org/broadinstitute/dsm/model/PDF/MRCoverPDF.java

```diff
@@ -0,0 +1,89 @@
+
+import lombok.NonNull;
+import org.apache.commons.lang3.BooleanUtils;
+import org.apache.commons.lang3.StringUtils;
+import org.broadinstitute.dsm.db.DDPInstance;
+import org.broadinstitute.dsm.db.InstanceSettings;
+import org.broadinstitute.dsm.db.MedicalRecord;
+import org.broadinstitute.dsm.db.dto.settings.InstanceSettingsDto;
+import org.broadinstitute.dsm.db.dto.user.UserDto;
+import org.broadinstitute.dsm.files.CoverPDFProcessor;
+import org.broadinstitute.dsm.files.PDFProcessor;
+import org.broadinstitute.dsm.files.RequestPDFProcessor;
+import org.broadinstitute.dsm.util.SystemUtil;
+import org.json.JSONObject;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+public class MRCoverPDF{
+    private final String JSON_START_DATE = "startDate";
+    private final String JSON_END_DATE = "endDate";
+    private DownloadPDF originalDownloadPDF;
+
+    public MRCoverPDF(@NonNull DownloadPDF downloadPDF){
+        this.originalDownloadPDF = downloadPDF;
+    }
+
+    public Map<String, Object> getValuesFromRequest(@NonNull String requestBody, DDPInstance ddpInstance, UserDto user) {
+        JSONObject jsonObject = new JSONObject(requestBody);
+        Set keySet = jsonObject.keySet();
+        String startDate = null;
+        String endDate = null;
+        if (keySet.contains(JSON_START_DATE)) {
+            startDate = (String) jsonObject.get(JSON_START_DATE);
+            if (!"0/0".equals(startDate) && !startDate.contains("/") && startDate.contains("-")) {
+                startDate = SystemUtil.changeDateFormat(SystemUtil.DATE_FORMAT, SystemUtil.US_DATE_FORMAT, startDate);
+            }
+            else if (StringUtils.isNotBlank(startDate) && startDate.startsWith("0/") && !startDate.equals("0/0")) {
+                startDate = "01/" + startDate.split("/")[1];
+            }
+        }
+        if (keySet.contains(JSON_END_DATE)) {
+            endDate = (String) jsonObject.get(JSON_END_DATE);
+            endDate = SystemUtil.changeDateFormat(SystemUtil.DATE_FORMAT, SystemUtil.US_DATE_FORMAT, endDate);
+        }
+        if (StringUtils.isBlank(originalDownloadPDF.getMedicalRecordId())) {
+            throw new RuntimeException("MedicalRecordID is missing. Can't create cover pdf");
+        }
+        //get information from db
+        MedicalRecord medicalRecord = MedicalRecord.getMedicalRecord(ddpInstance.getName(), originalDownloadPDF.getDdpParticipantId(), originalDownloadPDF.getMedicalRecordId());
+
+        Map<String, Object> valueMap = new HashMap<>();
+        //values same no matter from where participant/institution data comes from
+        valueMap.put(CoverPDFProcessor.FIELD_CONFIRMED_INSTITUTION_NAME, medicalRecord.getName());
+        valueMap.put(CoverPDFProcessor.FIELD_CONFIRMED_INSTITUTION_NAME_2, medicalRecord.getName() + ",");
+        valueMap.put(CoverPDFProcessor.FIELD_CONFIRMED_FAX, medicalRecord.getFax());
+        String today = new SimpleDateFormat("MM/dd/yyyy").format(new Date());
```

Comment:
for the today string see if we already have a method, if not move the date format string to static and up, so it is easier to find it if we need to make changes